radixdlt/src/main/java/com/radixdlt/consensus/SyncVerticesRPCRx.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.consensus;
+
+import com.radixdlt.consensus.VertexStore.GetVerticesRequest;
+import io.reactivex.rxjava3.core.Observable;
+
+/**
+ * Provider of GetVertices RPC request/response events
+ */
+public interface SyncVerticesRPCRx {
+
+	/**
+	 * Retrieve a never-ending stream of requests
```

Comment:
`requests` and `responses` are hot observables, right?

Reply:
no, cold observables but doesn't really matter

Reply:
not sure why this doesn't matter, isn't that relevant to know as a consumer of this interface?